freqtrade/optimize/optimize_reports/bt_output.py

```diff
@@ -233,8 +233,8 @@ def text_table_add_metrics(strat_results: Dict) -> str:
             ('Calmar', f"{strat_results['calmar']:.2f}" if 'calmar' in strat_results else 'N/A'),
             ('Profit factor', f'{strat_results["profit_factor"]:.2f}' if 'profit_factor'
                               in strat_results else 'N/A'),
-            ('Expectancy', f"{strat_results['expectancy']:.2f}" if 'expectancy'
-                           in strat_results else 'N/A'),
+            ('Expectancy Ratio', f"{strat_results['expectancy_ratio']:.2f}" if 'expectancy_ratio'
```

Comment:
is the output now "Expectancy Ratio" instead of  Expectancy - or was this labeled wrongly?

At any rate, we should update the corresponding backtesting.md page, too.
Maybe it also makes sense to output both values - so it's aligned to `/profit` ?

Reply:
Yes, I will make it print both as how it's in /profit